src/AddressValidation.php

```diff
@@ -67,6 +78,14 @@ public function __construct($accessKey = null, $userId = null, $password = null,
     }
 
     /**
+     * This message set the addressValidationResponseObject
+     * @param bool $value
+     */
+    public function validateReturnAVObject($value = true)
```

Comment:
Can we come up with a more prescriptive naming? Something like.. 'activateObjectReturn' or something? Something with activate and deactive would be great :)